config/environments/development.rb

```diff
@@ -56,10 +56,10 @@
   # config.only_path = true
 
 
-  config.after_initialize do
-    Bullet.enable = true
-    Bullet.bullet_logger = true
-    Bullet.raise = true
-  end
+  # config.after_initialize do
```

Comment:
Will we need this later or am I good to delete it?

Reply:
yes